### PR TITLE
Create JackMcRoberts.md

### PR DESCRIPTION
**Pull request by JackMcRoberts**
Add a bio for Jack McRpberts per Issue #14 

@JackMcRoberts Please validate information

### CODE REVIEW

File: docs/JackMcRoberts.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Jack McRoberts
```
Comment (mikercline) [inline docs/JackMcRoberts.md]: Maybe make the header more informative like:

# Jack McRoberts Bio

**Comment by mikercline** [comment]
Looks good to me, nicely done.

**Comment by JackMcRoberts** [comment]
Fixes #14

**Comment by JackMcRoberts** [comment]
Fixed discrepancies

**Comment by JackMcRoberts** [comment]
Unmerged commits

**Pull request merged**